website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,99 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
+
+To customize your optional configurations in dbt Cloud:
+
+- Click **Account home** from the left-hand side bar menu in dbt Cloud
+- From there, click **Project** and select your BigQuery project
+- Select your BigQuery project from the left-hand menu
+- Click **Settings** towards the top right and a **Project details** window will open
+- Scroll down to **Development Connection** and select BigQuery
+- Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configs you can set in dbt Cloud:
+
+<DetailsToggle alt_header="Priority">
+
+The `priority` for the BigQuery jobs that dbt executes can be configured with the `priority` configuration in your BigQuery profile. The priority field can be set to one of `batch` or `interactive`. For more information on query priority, consult the [BigQuery documentation](https://cloud.google.com/bigquery/docs/running-queries).
+
+</DetailsToggle>
+
+<DetailsToggle alt_header="Retries">
+
+Retries in BigQuery help to ensure that jobs complete successfully by trying again after temporary failures, making your operations more robust and reliable.
+
+</DetailsToggle>
+
+<DetailsToggle alt_header="Location">
+
+The `location` of BigQuery datasets can be set using the `location` setting in a BigQuery profile. As per the [BigQuery documentation](https://cloud.google.com/bigquery/docs/locations), `location` may be either a multi-regional location (for example, `EU`, `US`), or a regional location (like `us-west2`).
+
+</DetailsToggle>
+
+
+
+
+
+<DetailsToggle alt_header="Maximum bytes build">
+
+When a `maximum_bytes_billed` value is configured for a BigQuery profile, that allows you to limit how much data your query can process. It’s a safeguard to prevent your query from accidentally processing more data than you expect, which could lead to higher costs. Queries executed by dbt will fail if they exceed the configured maximum bytes threshhold. This configuration should be supplied as an integer number of bytes.
+
+If your `maximum_bytes_billed` is 1000000000, you would enter that value in the `maximum_bytes_billed` field in dbt cloud.
+
+|  config | info | type | example |
+| ------- | ----| ----- | -------- |
+| max bytes billed | info here | integer | 100000 |
+
+</DetailsToggle>
+
+
+
+
+<DetailsToggle alt_header="Execution project">
+
+By default, dbt will use the specified `project`/`database` as both:
+
+1. The location to materialize resources (models, seeds, snapshots, and so on), unless they specify a custom project/database config
+2. The GCP project that receives the bill for query costs or slot usage
+
+Optionally, you may specify an execution project to bill for query execution, instead of the project/database where you materialize most resources.
```

Comment:
```suggestion
Optionally, you may specify an execution project to bill for query execution, instead of the project/database where you materialize most resources.

</DetailsToggle>
```